doc/source/projects.rst

```diff
@@ -0,0 +1,75 @@
+Ray Projects (Experimental)
+===========================
+
+Ray projects make it easy to package a Ray application so it can be
+rerun later in the same environment. They allow for the sharing and
+reuse of existing code reliably.
+
+Quick start (CLI)
+-----------------
+
+.. code-block:: bash
+
+    # Creates a project in the current directory. It will create a
+    # project.yaml defining the code and environment and a cluster.yaml
```

Comment:
Is it possible to infer the env based on current conda installed packages?

Reply:
Not yet but should be, I created an issue here: https://github.com/ray-project/ray/issues/5487